packages/generators/generators/lib/plops/policy.js

```diff
@@ -15,14 +16,7 @@ module.exports = plop => {
       ...getDestinationPrompts('policy', plop.getDestBasePath(), { rootFolder: true }),
     ],
     actions(answers) {
-      let filePath;
-      if (answers.destination === 'api') {
-        filePath = `api/{{api}}`;
-      } else if (answers.destination === 'plugin') {
-        filePath = `plugins/{{plugin}}/server`;
-      } else {
-        filePath = `./`;
-      }
+      const filePath = getFilePath(answers.destination);
 
```

Comment:
You can't use `getFilePath` in this case.

The "default" value returned by `getFilePath` is `api/{{ id }}`, where as the policy default value is `./`.
Either you adapt the `getFilePath` method to have an argument to set a default value or you leave the code the way it was.

Both options are valid. I think that keeping the code as is might be the simplest as we don't add complexity to the `getFilePath` function for 2 use cases.